apps/protocol-api/src/main.ts

```diff
@@ -74,6 +74,7 @@ const permissions = shield(
       getUser: isAuthenticated,
       guild: or(isAuthenticated, hasToken),
       guilds: or(isAuthenticated, hasToken),
+      guildUsers: or(isAuthenticated, hasToken),
```

Comment:
Only users in the guild should be able to fetch other users in the guild. This will require a custom endpoint.

Reply:
We can handle this in another ticket